src/applications/mhv/secure-messaging/components/FoldersList.jsx

```diff
@@ -27,7 +27,7 @@ const FoldersList = props => {
                     folder.unreadCount > 0 &&
                     folder.id !== Folders.DRAFTS.id &&
                     `(${folder.unreadCount} unread messages)`}
-                </span>
+                </span>{' '}

```

Comment:
just checking if this is whitespace is needed

Reply:
You're right. This is not needed.